seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -6807,7 +6807,7 @@
             "Si la propiedad <code>prop</code> es <code>\"tracks\"</code> y <code>value</code> no está en blanco, empuja (<em>push</em>) el valor <code>value</code> al final del vector <code>tracks</code>.",
             "Si el valor <code>value</code> está en blanco, elimina esa <code>prop</code>.",
             "Siempre retorna el objeto <code>collection</code> entero.",
-            "<strong>Nota</strong><br>No olvides usar <code>notación corchete</code> cuando <a href=\"accessing-objects-properties-with-variables\" target=\"_blank\">accedes a propiedades de objetos con variables</a>."
+            "<strong>Nota</strong><br>No olvides usar <code>notación corchete</code> cuando <a href=\"javascript-algorithms-and-data-structures/basic-javascript/accessing-objects-properties-with-variables\" target=\"_blank\">accedes a propiedades de objetos con variables</a>."
```

Comment:
This line, too:
`javascript-algorithms-and-data-structures/basic-javascript/accessing-object-properties-with-variables`